SemanticAnalysis/parser.y

```diff
@@ -562,48 +771,6 @@ void default_value(int type) {
   }
 }
 
-
-/* ------------------- Handling Hash Tables --------------- */
-
-/* ------Return Stack------ */
-
-
-/* ------While Stack------ */

```

Comment:
Why did you remove while stack?????

Reply:
While stack functions are defined twice(top_while, pop_while,.. all those). So i removed one of those.

Reply:
So, you mean it is already there in your latest commit?
If that is the case, then fine.